beginner/webserver/solution.md

```diff
@@ -7,15 +7,15 @@ docker build -t "webserver:latest" .
 
 Run the docker container with "docker run":  
 ```
-docker run -p 8000:80 -d webserver
+docker run -d --name nginx-webserver -p 8000:8080 webserver:latest
```

Comment:
please also change 8000 to something else. 8 and 0 look very alike

Reply:
What a trap, I thought it is 8080 already. ^^
I will change it.